pyproject.toml

```
@@ -1,9 +1,11 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools",
```

```suggestion
    "setuptools>=42",
```

Before 42, setuptools had buggy pyproject.toml support (and no pyproject.toml support before 40), so I'd always put a lower limit here.